namesys/interface.go

```diff
@@ -47,6 +47,14 @@ const (
 	// trust resolution to eventually complete and can't put an upper
 	// limit on how many steps it will take.
 	UnlimitedDepth = 0
+
+	// ImmutableTTL is the time-to-live value to be reported for immutable
+	// paths.
```

Comment:
actually, no, this should return a special value which is infinity -- immutable things are meant to be cached forever_. The http gateway can translate to the stupid http "forever means a year" thing.